h5py/_hl/vds.py

```diff
@@ -102,6 +103,20 @@ def shape(self):
     def __getitem__(self, key):
         tmp = copy(self)
         tmp.sel = select(self.shape, key, dataset=None)
+        try:
+            key = tuple(key)
+        except TypeError:
+            key = (key,)
```

Comment:
```suggestion
        key = key if isinstance(key, tuple) else (key,)
```

The `isinstance()` check matches what we do in e.g. `Dataset.__getitem__`.

Reply:
Good call, thanks!